libp2p/NodeTable.cpp

```diff
@@ -375,10 +376,6 @@ void NodeTable::noteActiveNode(shared_ptr<NodeEntry> _nodeEntry, bi::udp::endpoi
         return;
 
     LOG(m_logger) << "Active node " << *_nodeEntry;
-    // TODO: don't activate in case endpoint has changed
-    _nodeEntry->endpoint.setAddress(_endpoint.address());
```

Comment:
This part is basically moved to Pong handler (lines 512-516), because we should trust changed endpoint only after valid Pong